### ISSUE TITLE
Perform accessibility enhancements

### ISSUE THREAD

**Issue by talitapagani** [wejs/we#112]
According to the Accessibility Evaluation conducted at We.js website, taking into account WCAG 2.0 guidelines, the following adjustments need to be performed:

## WCAG 2.0 Level A
- [ ] Add `lang='en'` attribute to the `<html>` tag. It's needed to inform to screen readers the natural language of the website content;
- [ ] Add and make revision of alternative (ALT) text in images. The ALT text should be relevant, concise and represent the image content or meaning that is relevant to the context;
- [ ] Add missing attribute `for` to form `<label>`, specially at Login and Register pages;
- [ ] Add a unique `<title>` for each We.js page, following W3C/WAI guidelines;
- [ ] Add a _skip to content_ link at the beginning of pages. This link skip the navigation menu and header, jumping to the main content;
- [ ] Add ARIA landmark roles for header, search, menu, footer and aside content.

## WCAG 2.0 Level AA
- [ ] Add heading titles (`<h1>`) for pages that are missing them, such as Plugins and Themes;
- [ ] Make focus visible for form fields;
- [ ] Improve error messages, specially on form pages;
- [ ] Add ARIA roles, states and properties to page elements;
- [ ] Improve color contrast between elements background and foreground colors;
- [ ] Provide a high contrast mode.

**Comment by albertosouza** [comment]
:+1:

**Comment by albertosouza** [comment]
i am working with a new image selector for solve: *Add and make revision of alternative (ALT) text in images. The ALT text should be relevant, concise and represent the image content or meaning that is relevant to the context*

**Comment by albertosouza** [comment]
@talitapagani Accessibility Evaluation Report 1 online and avaible in: http://wejs.org/public/project/aer-1/public-website-of-wejs-framework-and-wejs-events-evaluation-report.html

**Comment by talitapagani** [comment]
@albertosouza Thanks! I'm going to work first on the other items related to Level A compliance.

**Comment by albertosouza** [comment]
Hi @talitapagani , you can send an example of accessibility menu?
Like this go to content link.

**Comment by albertosouza** [comment]
Skip content on in we.js site for cluster theme: 
![Skip content link example in wejs.org site](http://wejs.org/api/v1/image/original/1456151168403_36d2fe30-d970-11e5-aee5-574b3f08efe4.png)

**Comment by aw76838434** [comment]
very good good
